src/dbt/kipptaf/models/extracts/google/sheets/rpt_gsheets__stipends_app_roster.sql

```diff
@@ -0,0 +1,118 @@
+                then concat(report_to_sam_account_name, '@apps.teamschools.org')
+                when business_unit_home_name like '%Miami%'
+                then concat(report_to_sam_account_name, '@kippmiami.org')
+            end as second_approver_google,
+
+            case
+                when department_home_name = 'Operations'
+                then 'Operations'
+                when department_home_name = 'School Leadership'
+                then 'Instructional'
+                when business_unit_home_name like '%Family%'
+                then 'CMO'
+                else 'Regional'
+            end as route
+        from {{ ref("base_people__staff_roster") }}
+        where
+            job_title in (
+                'Director School Operations',
+                'Director Campus Operations',
+                'Director',
+                'School Leader',
+                'School Leader in Residence'
+            )
+            and assignment_status != 'Terminated'
+    ),
+
+    roster as (
+        select
+            sr.employee_number,
+            sr.payroll_group_code,
+            sr.worker_id,
+            sr.payroll_file_number as file_number,
+            sr.position_id,
+            sr.job_title,
+            sr.home_work_location_name as location,
+            sr.department_home_name as department,
+            sr.preferred_name_lastfirst as preferred_name,
+            sr.user_principal_name as email,
+            sr.google_email,
+            sr.assignment_status as status,
+            sr.business_unit_home_name as region,
+            sr.worker_termination_date,
+            coalesce(
+                sr.home_work_location_abbreviation, sr.home_work_location_name
+            ) as location_abbr,
+            case
+                when
+                    sr.home_work_location_name not like '%Room%'
+                    and sr.department_home_name in ('Operations', 'School Support')
+                then 'Operations'
+                when
+                    sr.home_work_location_name not like '%Room%'
+                    and sr.business_unit_home_name not like '%Family%'
+                then 'Instructional'
+                when
+                    sr.home_work_location_name like '%Room%'
+                    and sr.business_unit_home_name not like '%Family%'
+                then 'Special'
+                when
+                    sr.home_work_location_name like '%Room%'
+                    and sr.business_unit_home_name like '%Family%'
+                then 'CMO'
+                else 'Regional'
+            end as route,
+            coalesce(cc.name, sr.home_work_location_name) as campus,
+        from {{ ref("base_people__staff_roster") }} as sr
+        left join
+            {{ ref("stg_people__campus_crosswalk") }} as cc
+            on sr.home_work_location_name = cc.location_name
+        where
+            sr.worker_termination_date is null
+            or sr.worker_termination_date > '2023-07-01'
```

Comment:
should this be dyanamic for the current year?

Reply:
i'm going to have to change it every quarter, we can leave as is for now